client/app/shared/ci-helper/ci-helper.controller.js

```diff
@@ -455,6 +461,8 @@ const CiHelperController = function CiHelperController(
         if (!config || config.isActive) { return; }
         const parentLauncherId = vm.activeLauncher.parentLauncherId || vm.activeLauncher.id;
 
+        delete vm.activeLauncher.providerId;
```

Comment:
Please use `Reflect.deleteProperty()` instead of `delete`

Reply:
fixed